pkg/api/config_test.go

```diff
@@ -0,0 +1,108 @@
+package api_test
+
+import (
+	"regexp"
+	"strings"
+	"testing"
+)
+
+var searchPattern = regexp.MustCompile(`(\d+)`)
+var replacePattern = []byte(` $1 `)
+
+// padNumbers inserts spaces around numbers within a string
+func padNumbers(s string) string {
+	b := []byte(s)
+	b = searchPattern.ReplaceAll(b, replacePattern)
+	return string(b)
+}
+
+func TestPadNumbers(t *testing.T) {
+	testcases := []struct {
+		input    string
+		expected string
+	}{
+		{input: "single123number", expected: "single 123 number",},
+		{input: "single123numbers345spread", expected: "single 123 numbers 345 spread",},
+		{input: "number_at_end234", expected: "number_at_end 234 ",},
+		{input: "123number_at_start", expected: " 123 number_at_start",},
+		{input: "123number_at_start", expected: " 123 number_at_start",},
+	}
+
+	for _, test := range testcases {
+		got := padNumbers(test.input)
+		if got != test.expected {
+			t.Errorf("padNumbers(%v) != %v, expected %v", test.input, got, test.expected)
+		}
+	}
+}
+
+// Converts a string to lower camel case
```

Comment:
wouldn't it be a good idea to move the functional code out into a separate file from the start?

Reply:
this function will only be used from another testcase which will be within this file. That is why this is currently just in config_test.go. already discussed this with @jim-minter